src/chess/uciloop.cc

```diff
@@ -200,6 +200,11 @@ void UciLoop::SendResponse(const std::string& response) {
   std::cout << response << std::endl;
 }
 
+void UciLoop::SendId() {
+  SendResponse("id name The Lc0 chess engine. v0.13");
```

Comment:
I kind of would like to change the version string to be xx.yy.zz  major/minor/patch.  I have server side change pending to accept either.

I would just suggest one thing here, move the version into it's own var (maybe file)
static const version[] = "v0.13"  or "0.13.0"

SendResponse(StrCat("id name..", version));

Reply:
Hmm also can you print the version at startup like:
https://github.com/LeelaChessZero/lc0/pull/64/files

Or will that screw up uci engines?  We need the version in self-play mode as well.

Reply:
I centralised the id printing to this one place, and made self play mode invoke it as well.

PR 64 is simpler and probably fine.

The server side shouldn't need a change to accept either I would have thought, the version parser should parse either already?

Reply:
I'd also suggest to put version into a separate file, something like `version.inc` (which would be included into .cc file but is not really a header).
When we switch to auto-push of a new version, this file will be automatically generated, so it should be simple.

Also, I think we should switch to "v0.13.00" format of the version.

Reply:
This comment was addressed. Commenting to convince github of that.